install.sh

```diff
@@ -282,6 +282,12 @@ export LD_LIBRARY_PATH="${CREW_PREFIX}/lib${LIB_SUFFIX}:/lib${LIB_SUFFIX}"
 
 echo "export CREW_PREFIX=${CREW_PREFIX}" >> "${CREW_PREFIX}/etc/env.d/profile"
 
+# Install activesupport gem for ruby
+echo_info 'Installing essential ruby gems.'
+gem update -N --system
+gem install -N activesupport --conservative
+gem install -N concurrent-ruby --conservative
```

Comment:
Doesn't this assume Ruby is installed on the host machine?

Reply:
We've got crew at this point, so we've got ruby.

Reply:
This is just before we run crew, so yes we have ruby at this point. This PR basically adds the deps necessary to get gem updating working as well at this point.